docs/gettingstarted/quickstart.ipynb

```diff
@@ -116,13 +947,281 @@
+      "text/plain": [
+       "<Figure size 500x400 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "image/png": {
+       "height": 408,
+       "width": 513
+      }
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax = plt.subplots(nrows=1, ncols=1, figsize=(5,4))\n",
```

Comment:
Can we add a title and error bars for this plot?

In general it's also difficult for me to figure out what this plot means but that might be beyond this PR (the choice of dataset and structure function might not be great)

Reply:
I'm not sure what would be the error of the SF. Perhaps @nevencaplar would have some ideas on this 😄